packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -100,12 +101,15 @@ export class WorkspaceContextUtil {
       try {
         const connection = await this.getConnection();
         this._orgId = connection?.getAuthInfoFields().orgId;
-      } catch (error) {
+      } catch (error: unknown) {
         this._orgId = '';
-        console.log(
-          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          `There was an problem getting the orgId of the default org: ${error}`
-        );
+        if(error instanceof Error) {
+          console.log(
+            'There was an problem getting the orgId of the default org: '
+            , error
+          );
+          TelemetryService.getInstance().sendException(WORKSPACE_CONTEXT_ORG_ID_ERROR, `name: ${error.name}, message: ${error.message}`);
```

Comment:
I wanted to ensure that we could identify these errors so I gave it a unique name instead of using the name on the error.